include/crm/crm.h

```diff
@@ -63,7 +63,7 @@ extern "C" {
  * >=3.2.0:  DC supports PCMK_EXEC_INVALID and PCMK_EXEC_NOT_CONNECTED
  * >=3.19.0: DC supports PCMK__CIB_REQUEST_COMMIT_TRANSACT
  */
-#define CRM_FEATURE_SET "3.20.3"
+#define CRM_FEATURE_SET "3.20.4"
```

Comment:
Once this PR gets merged, I think it's worth a little bit of publicity - update https://projects.clusterlabs.org/w/projects/pacemaker/pacemaker_feature_set/ to mention that all command line tools now support formatted output.

Reply:
Yeah, but I don't care about those nearly as much - they're presumably going to get rewritten one day (you should see Ken's in-progress branch that was dealing with crm_report).  I still think it's worth hyping it up a bit.